tsdb/shard.go

```diff
@@ -58,6 +58,12 @@ var (
 	ErrShardDisabled = errors.New("shard is disabled")
 )
 
+var (
+	// Static objects to prevent small allocs.
+	staticTimeByteSlice = []byte("time")
```

Comment:
Can you think of a better name for `staticTimeByteSlice`? There isn't much static about a variable 😄 .

How about `timeTag`?